health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/UpdateChecklist.js

```
@@ -398,12 +401,19 @@ const UpdateChecklist = () => {
           let checklistTypeTemp = checklistType.toUpperCase().replace(/ /g, "_");
           if (checklistTypeCode) checklistTypeTemp = checklistTypeCode;
           let roleTemp = role.toUpperCase().replace(/ /g, "_");
+          let helpTextCode = helpText.toUpperCase().replace(/ /g, "_");
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'helpTextCode' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)